DataFormats/L1THGCal/interface/HGCalClusterT.h

```diff
@@ -1,211 +1,284 @@
 #ifndef DataFormats_L1Trigger_HGCalClusterT_h
 #define DataFormats_L1Trigger_HGCalClusterT_h
 
+/* CMSSW */
 #include "DataFormats/Common/interface/Ptr.h"
 #include "DataFormats/Common/interface/PtrVector.h"
 #include "DataFormats/GeometryVector/interface/GlobalPoint.h"
 #include "DataFormats/L1Trigger/interface/L1Candidate.h"
 #include "DataFormats/L1THGCal/interface/HGCalTriggerCell.h"
 #include "DataFormats/L1THGCal/interface/ClusterShapes.h"
-#include "Math/Vector3D.h"
 
+/* ROOT */
+#include "Math/Vector3D.h"
 
 namespace l1t 
 {
   template <class C> class HGCalClusterT : public L1Candidate 
   {
 
-    public:
```

Comment:
The indentations in this file are still modified everywhere compared to before; it is impossible to see what has changed and what didn't change.